src/components/Item/ItemInfo.js

```diff
@@ -80,25 +80,26 @@ function ItemInfo(props) {
       </Grid>
       <Grid item xs={4}>
         <Grid container alignItems="center" direction="column">
-          {currentUser.id === itemDetail.ownerId ? (
-            <Button
-              onClick={editPosting}
-              // startIcon={<EditIcon />}
-              variant="outlined"
-              color="primary"
-            >
-              Edit Posting
-            </Button>
-          ) : (
+          {!currentUser || currentUser._id !== itemDetail.ownerId ? (
             <Button
               variant="contained"
               color="primary"
               onClick={() => {
                 props.openOfferingModal();
               }}
+              disabled={currentUser === null}
```

Comment:
I don't think currentUser is ever null? It's usually currentUser.user that can be set to null if not logged in. currentUser is default to {isFetching: ... , isFailed: ... , user: ....}

Reply:
In my mapStateToProps, I setting current user to state.currentUser.user because it was a faster fix than finding all references to currentUser

Reply:
Makes sense. So the variable "currentUser" is actually "currentUser.user". Will mark this conv as resolved